### ISSUE TITLE
Managed Dependency Check Failed

### ISSUE THREAD

**Issue by abinaya-95** [GoogleCloudPlatform/cloud-code-vscode#434]
**Environment**:
*(Hint: "Report Extension Issue on Github" command will fill these out for you.)*

**Version information**

Cloud Code Extension version:

VSCode version:

OS: 

Cloud SDK:

Skaffold: 

Kubectl:

**Description**:

**Repro step**:

**Comment by sivakku** [comment]
Please reopen the bug with details filled out

**Comment by manterfield** [comment]
@sivakku 
I'd guess this and the other issue came from both users pressing 'report issue' on this:
<img width="708" alt="Screenshot 2021-06-11 at 13 08 25" src="https://user-images.githubusercontent.com/3941150/121684052-1cbdee00-cab6-11eb-8d0c-122b18f09afd.png">

I just started getting this message too when trying to deploy to Cloud Run. Running on MacOS, skaffold is installed and both the cloud-code version and my global one seem to be working fine when called directly.

I just tried myself, the template created suggests we can autopopulate the environment data by running "Cloud Code: Report Extension Issue on Github" command. Unfortunately that command fails too with:

"Command 'Cloud Code: Report Extension Issue on Github' resulted in an error (Running the contributed command: 'cloudcode.reportIssueOnGithub' failed.)"
